cmd/generate/config/rules/config.tmpl

```diff
@@ -63,23 +63,23 @@ regexes = [
     "{{ $stopword }}",{{ end }}
 ]{{ end }}
 {{ end }}
-{{- if and $rule.Verify.URL $rule.Verify.HTTPVerb }}[rules.Verify]
-url = "{{ $rule.Verify.URL }}"
+{{- if and $rule.Verify }}[rules.Verify]
```

Comment:
```suggestion
{{- if and $rule.Verify }}[rules.verify]
```

Not sure if this is the best practices, but dropping `verify` to lowercase keeps it consistent with `allowlist`. WDYT?

Reply:
Consistency is nice; I can see this driving someone crazy.